charts/velero/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.2.0
 description: A Helm chart for velero
 name: velero
-version: 2.8.0
+version: 2.8.1
```

can you please bump the version, 2.8.1 is already in master

```suggestion
version: 2.8.2
```

---

Version bumped